vlib/v/checker/tests/go_expr.out

```diff
@@ -1,4 +1,4 @@
-vlib/v/checker/tests/go_expr.vv:2:5: error: expression in `go` must be a function call
+vlib/v/checker/tests/go_expr.vv:2:5: error: expression in `spawn` must be a function call
```

Comment:
The code in the .vv file should be changed too to use the `spawn` keyword, otherwise the error message text is mismatched to it.